services/121-service/src/guards/authenticated-user.decorator.ts

```diff
@@ -9,10 +10,20 @@ export interface AuthenticatedUserParameters {
   readonly isGuarded?: boolean;
 }
 
-export const AuthenticatedUser = (
-  parameters?: AuthenticatedUserParameters,
-): ReturnType<typeof SetMetadata> =>
-  SetMetadata('authenticationParameters', {
-    ...parameters,
-    isGuarded: true,
-  });
+export const AuthenticatedUser = (parameters?: AuthenticatedUserParameters) => {
+  return applyDecorators(
+    SetMetadata('authenticationParameters', {
+      ...parameters,
+      isGuarded: true,
+    }),
+    ApiResponse({
```

Comment:
Should we not remove the "status: HttpStatus.FORBIDDEN", and "status: HttpStatus.UNAUTHORIZED" API responses from all endpoints that use the AutenticatedUser decorator?

Reply:
Ah yes thanks! I forgot to remove HttpStatus.UNAUTHORIZED because I added that one later

Reply:
Still found FORBIDDEN in: services/121-service/src/financial-service-providers/financial-service-provider.controller.ts

Reply:
Ah I originally left that one because it had a different text. But I see FORBDDEN is never called in the service so I removed it